util/make_dir/config.nci

```diff
@@ -1,5 +1,5 @@
 module purge
-module load intel-fc/17.0.1.132
-module load intel-cc/17.0.1.132
-module load netcdf/4.3.2
-module load openmpi/1.10.2
+module load intel-compiler/2019.5.281
+module load netcdf/4.7.1
+module load openmpi/4.0.1
+
```

Comment:
these changes to `config.nci` are unnecessary (but harmless), as `config.gadi` is used in the gadi-transition branch

Reply:
Noted.

Reply:
Addressed by commit 861be6d